src/app/backend/integration/handler.go

```diff
@@ -0,0 +1,37 @@
+package integration
+
+import (
+	"net/http"
+
+	"github.com/emicklei/go-restful"
+	"github.com/kubernetes/dashboard/src/app/backend/integration/api"
+)
+
+// IntegrationHandler manages all endpoints related to integrated applications, such as state.
+type IntegrationHandler struct {
+	manager IntegrationManager
+}
+
+// Install creates new endpoints for integrations.
```

Comment:
I'll describe it better. Right now it's very simple and just exposes integration state but other integrations might want to install their own endpoints also.